src/expressions.rst

```diff
@@ -4813,3 +4884,25 @@ within the :t:`capturing expression`, as follows:
 :dp:`fls_wvob7114tfat`
 A tool selects the first :t:`capture mode` that is compatible with the use of
 the :t:`capture target`.
+
+.. _fls_ZfIBiJMf8qE1:
+
+Arithmetic Overflow
+-------------------
+
+:dp:`fls_oFIRXBPXu6Zv`
+An :dt:`arithmetic overflow` occurs if an :t:`arithmetic expression` or a
+:t:`negation expression` computes a :t:`value` of a :t:`scalar type` that lies
+outside of the range of valid :t:`[value]s` for the :t:`scalar type`.
+
+.. rubric:: Dynamic Semantics
+
+:dp:`fls_8fU3h8PLasqA`
+There are two allowed behaviors for :t:`arithmetic overflow`:
+
+#. :dp:`fls_R48VKcEIbfXC`
+   Evaluation of the expression may result in a :t:`panic`.
+
+#. :dp:`fls_QMpI8K43K2yU`
+   The resulting value of the expression may be truncated, discarding the most
```

Comment:
The resulting :t:`value` of the :t:`expression` may be truncated, discarding the most significant bits that do not fit in the target :t:`type`.